.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: CI
+
+# Run the deployment only when code is committed to the branch.
+on: pull_request
+
+# Cancels all previous workflow runs for pull requests that have not completed.
+concurrency:
+    # The concurrency group contains the workflow name and the branch name for pull requests
+    # or the commit hash for any other events.
+    group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+    cancel-in-progress: true
+
+jobs:
+  lint-and-audit:
+
+    runs-on: self-hosted
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.PRIVATE_ACCESS_TOKEN }}
+
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v2.5.1
+      with:
+        node-version: 12.x
+        cache: 'npm'
+
+    - name: Set composer token
+      run: composer config github-oauth.github.com ${{ secrets.PRIVATE_ACCESS_TOKEN }} 
+
+    - name: Composer Install
+      uses: ramsey/composer-install@v1
+
+    - name: NPM Dependencies
+      if: always()
+      run: npm install
```

Comment:
This can be removed since we don't have npm dependencies

Reply:
Done